ngraph/core/include/ngraph/op/hswish.hpp

```diff
@@ -0,0 +1,55 @@
+//*****************************************************************************
+// Copyright 2017-2020 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#pragma once
+
+#include "ngraph/node.hpp"
+#include "ngraph/op/op.hpp"
+
+namespace ngraph
+{
+    namespace op
+    {
+        namespace v4
+        {
+            /// \brief A HSwish Activation Function
+            /// f(x) =  x * min(max(x + 3, 0), 6) / 6 or
+            /// f(x) = x * min(ReLU(x + 3), 6) / 6
+            ///
+            class NGRAPH_API HSwish : public ngraph::op::Op
```

Comment:
IMO `HSwish` can inherit from `ngraph::op::util::UnaryElementwiseArithmetic`. It provides some additional checks (for example block user to pass boolean element type) and provides shape inference (validate_and_infer_types will be not needed).
Looks for example for `Relu`.

Reply:
I thought it's deprecated, but seems legit. 
Thanks, done.